Makefile

```diff
@@ -37,7 +37,7 @@ help:
 .PHONY: services
 services: args?=up -d
 services: python
-	@tox -qe docker-compose -- $(args)
+	@tox -qe dockercompose -- $(args)
```

Comment:
See https://github.com/hypothesis/lms/pull/1956#discussion_r462210298 for the renaming of this